.github/workflows/publish.yml

```diff
@@ -42,8 +40,6 @@ jobs:
       - name: Setup CodeQL
         if: steps.check_version.outputs.publish == 'true'
         uses: ./.github/actions/install-codeql
-        with:
-          codeql-cli-version: ${{ env.CODEQL_CLI_VERSION }}
```

Comment:
Also remove the with `codeql-cli-version` from the other uses of `install-codeql`.